_episodes/41-code-review.md

```diff
@@ -278,127 +269,140 @@ levels of documentation?
 * Are there clear and useful comments that explain complex designs
 and focus on the "why/because" rather than the "what/how"?
 
-### Adding a review comment
-
-To add comments to a review, you need to open up the pull request.
-Then go to the `Files changed` tab.
-![The files changed tab of a pull request](../fig/github-pull-request-files-changed.png){: .image-with-shadow }
-When you find a line that you want to add a comment to, click on the
-plus (+) icon next to line.
-![Adding a review comment to a pull request](../fig/github-pull-request-add-comment.png){: .image-with-shadow }
+#### Things Not to Look for in a Code Review
+
+The overriding priority for reviewing code should be making sure progress is being made -
+don't let perfect be the enemy of good here.
+According to [“Best Kept Secrets of Peer Code Review” (Cohen, 2006)](https://www.amazon.co.uk/Best-Kept-Secrets-Peer-Review/dp/1599160676)
+the first hour of reviewing code is the most effective, with diminishing returns after that.
+
+To that end, here are a few things you *shouldn't* be trying to spot when reviewing:
+* Linting issues, or anything else that an automated tool can spot - get the Continuous Integration (CI) to do it.
+* Bugs - instead make sure there are tests for all cases.
+* Issues that pre-date the change - raise separate PRs fixing these issues separately to avoid heading down a rabbit hole.
+* Architecture re-writes - try to have design discussions upfront, 
+or else have a meeting to decide whether the code needs to be rewritten.
+
+#### Adding a review comment
+
+1. To be able to add your review to a pull request created on your course mate's repository, 
```

Comment:
I recall we discussed a callout about the general case of code review where you don't strictly need to add people to your repo to do a code review, which should be added.

Reply:
Looks like I failed to commit but did a push which did not have any affect on my latest changes - sorry @steve-crouch